recipes/creme/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "creme" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/creme-ml/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "109e308070e023825efe0b3eb6f94898e59a24ce52da259828b2eb473dfed4c6"
+
+build:
+  skip: True  # [win or py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.16.4
```

Comment:
Do you need most of these in `host`? It looks like `python` and `pip` will be enough (these are only required to run the `setup.py` script).